ansible/roles/ocp4-workload-idm/tasks/remove_workload.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Implement your Workload removal tasks here
+
+# Find IP of bastion
+- name: Gather VPC facts
+  ec2_vpc_net_facts:
+    filters:
+      tag:Name: "{{ aws_vpc_name }}"
```

Comment:
where is `aws_vpc_name` defined ?

Reply:
found it: `roles/infra-ec2-template-generate/defaults/main.yml:aws_vpc_name: "{{ subdomain_base }}"`